### DISCUSSION TITLE
what to do when there's update from win11?

### DISCUSSION THREAD

**Discussion by blecaplol** [valinet/ExplorerPatcher#1456] [Q&A]
What is the proper way to revert back before anything messes up or something? sorry im noob at this. i dont wanna get caught and have to reinstall the entire OS :(

**Comment by Evengard** [comment]
The app should be smart enough for you to not worry about that. Just enable the app autoupdate feature. In the worst case scenario you'd need to restart your PC one more time, I guess.

**Reply by blecaplol** [reply]
> The app should be smart enough for you to not worry about that. Just enable the app autoupdate feature. In the worst case scenario you'd need to restart your PC one more time, I guess.

ok! sorry can u tell me where to enable the app autoupdate cant seem to identify it properly.

**Reply by Evengard** [reply]
![picture](https://github.com/valinet/ExplorerPatcher/assets/798735/1dc1f614-bdc1-43e0-9c8d-395001577c49)
Something like that

**Comment by pyrates999** [comment]
You'll get a notification from windows when there is an update available for explorer patcher.

To install updates:

1. Open it up by right clicking on the taskbar, selecting properties
2. Go to updates
3. Click on **Update program and restart File Explorer**
4. If you want to manually check for updates, click on **Check for updates** and check the notification from windows

**Comment by blecaplol** [comment]
Cheers all!
